### PR TITLE
feat: add contribution manifest

### PR DESCRIPTION
**Pull request by jonathan-mayer**
## Motivation

We didn't have a contribution manifest until now.

## Changes

Added contribution manifest

## TODO

- [x] I've assigned myself to this PR

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+`type: description message`
+
+### Types
+
+| Type     | Example                         | Description                                                                          |
+| -------- | ------------------------------- | ------------------------------------------------------------------------------------ |
+| feat     | feat: add more workload types   | A new feature                                                                        |
+| fix      | fix: stop tests from failing    | A fix of a bug                                                                       |
+| chore    | chore: rm unneeded consts       | An annoying task, that has/had to be done                                            |
+| refactor | refactor: improved error logs   | Change that don't change any functionality                                           |
```
Comment (JTaeuber) [inline CONTRIBUTING.md:26]: It's either doesn't or you could write changes

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+### Common Abbreviations
+
+| Abbreviation | Meaning |
+| ------------ | ------- |
+| sw           | switch  |
+| rm           | remove  |
+
+## Code and Structure
+
+Try to stick to golang best practices and standarts.
```
Comment (JTaeuber) [inline CONTRIBUTING.md:41]: standards

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+Package structure standards can be found here: https://github.com/golang-standards/project-layout
+
+## Issues
+
+If applicable use the issue template. This ensures a consistent structure which makes it easier to find important details.
+Issues which aren't ready for processing, you can mark them as a draft by writing "Draft: " infront of the issue name.
```
Comment (JTaeuber) [inline CONTRIBUTING.md:47]: can be marked

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+
+## Pull Requests
+
+Use the Pull Request Template. This ensures a consistent structure which makes it easier to find important details.
+Set yourself and any other collaborators as assignee.
+
+## Pre Commit
+
+It is recommended to install pre-commit. This insures that formatting is consistent, you don't commit to protected branches and you don't accedentally commit broken code or new functionality without changing the tests. The installation process is in the [README](README.md#setting-up-pre-commit)
```
Comment (JTaeuber) [inline CONTRIBUTING.md:56]: accidentally

**Review by JTaeuber — CHANGES_REQUESTED** [review]

**Review by JTaeuber — APPROVED** [review]
LGTM

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+`type: description message`
+
+### Types
+
+| Type     | Example                         | Description                                                                          |
+| -------- | ------------------------------- | ------------------------------------------------------------------------------------ |
+| feat     | feat: add more workload types   | A new feature                                                                        |
+| fix      | fix: stop tests from failing    | A fix of a bug                                                                       |
+| chore    | chore: rm unneeded consts       | An annoying task, that has/had to be done                                            |
+| refactor | refactor: improved error logs   | Changes that don't change any functionality                                           |
```
Comment (jonathan-mayer) [inline CONTRIBUTING.md:26]: table formatting is off by 1

**Pull request merged**